### DISCUSSION TITLE
The recommendations should be the defaults.

### DISCUSSION THREAD

**Discussion by beedell-roke** [FastForwardTeam/FastForward#1382] [Ideas]
I always configure the recommendations to be my configuration. I'm surprised that what the author recommends isn't the default.

**Comment by undeadjess** [comment]
We are still working or making the tracker bypassing work again, so its not on by default at the moment.
Mozilla don't want us to have crowd bypass on by default, quoting privacy concerns.

As soon as the tracker bypassing is fixed, I'll go through and improve the defaults.

**Reply by beedell-roke** [reply]
> As soon as the tracker bypassing is fixed, I'll go through and improve the defaults.

@undeadjess, has the work commenced? I don't mind filing an issue to track it.

**Reply by undeadjess** [reply]
FastForward is no longer being actively maintained, and the fix for tracker bypassing was never completed... iirc the api we were using to fetch where the trackers redirected was deprecated, and this functionality would require us to rebuild, maintain, and host it ourselves - we couldn't find any public apis providing the same functionality.

It would be relatively simple to make and it's something I'm vaguely interested in doing anyway, but I simply won't have the time for another few weeks at least. I'll set myself a reminder in a few weeks and give updates on this thread. If I decide to go ahead with it, expect a new repository for the api.

If its something you would be interested in helping with, feel free to reach out to me via [email](mailto:down-swan-undying@duck.com) or the [FastForward Discord server](https://discord.gg/RSAf7b5njt)

**Comment by beedell-roke** [comment]
Originally, due to https://github.com/orgs/community/discussions/142158#discussioncomment-14973339, I posted https://canary.discord.com/channels/876622516607656006/876622517115174934/1297908659291947061:

> I received an e-mail response to a GitHub discussion - https://github.com/FastForwardTeam/FastForward/discussions/1382#discussioncomment-8968735 - but it appears that that entire section has been removed, so I can't respond. I think that's a bad idea - couldn't the repository owner merely prevent anyone from *opening* discussions? Otherwise, all that content is... inaccessible.

However, I've reposted this at https://github.com/FastForwardTeam/FastForward/discussions/1382#discussioncomment-14973355, not least due to https://stackoverflow.com/revisions/38333273/1#:~:text=GitHub%20treats%20comments%20received%20via,not%20permitted%20in%20email%20replies. Therefore, please hide this [^1] as a duplicate.

[^1]: https://github.com/orgs/community/discussions/59323#discussion-5339628
